lib/SimpleSAML/Metadata/SAMLParser.php

```diff
@@ -333,10 +334,6 @@ public static function parseDescriptorsFile($file)
             throw new \Exception('Failed to read XML from file: '.$file);
         }
 
-        if ($doc->documentElement === null) {
-            throw new \Exception('Opened file is not an XML document: '.$file);
```

Comment:
Why is this removed? Can't `documentElement` be `null`?

Reply:
Nope, it's always a DOMElement:
https://www.php.net/manual/en/class.domdocument.php#domdocument.props.documentelement

Reply:
Great! 👍